starter-code/pizza.js

```diff
@@ -1 +1,136 @@
 // Write your Pizza Builder JavaScript in this file.
+
+// SELECTION
+
+  // FONCTION ADD / DELETE
+
+function addDelete (ingredient){
```

Comment:
This function is cool! It great that you turned this into a function rather than repeating lots of code